ReactWindows/ReactNative.Shared/UIManager/BorderedCanvasManager.cs

```diff
@@ -14,14 +15,52 @@ namespace ReactNative.UIManager
     public abstract class BorderedCanvasManager<TCanvas> : BorderedViewParentManager<TCanvas>
         where TCanvas : Canvas
     {
+        protected override bool HasBorder(TCanvas view)
+        {
+            return GetBorder(view) != null;
+        }
+
+        private Border GetBorder(TCanvas view)
```

Comment:
One approach might be to subclass Canvas and maintain the state about whether the first child is a built-in Border there.

---
In reply to: [127780765](https://github.com/rozele/react-native-windows/pull/1#discussion_r127780765) [](ancestors = 127780765)